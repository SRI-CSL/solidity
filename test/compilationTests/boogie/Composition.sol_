pragma solidity ^0.4.23;

contract A {
    uint public x;
}

contract B {
    A a;

    function f() public returns (uint) {
        return a.x();
    }
}