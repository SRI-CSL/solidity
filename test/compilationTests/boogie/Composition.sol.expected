Verifying A::someFunc: OK
Verifying B::callOtherFunc: OK
Verifying B::accessOtherMember: OK
No errors found.
